docs/content/en/integrations/parsers/file/coverity_scan.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Coverity Scan JSON Report"
+toc_hide: true
+---
+Run `coverity scan --project-dir <project_dir> --local <result_file> --local-format json` to create the JSON report.
```

Comment:
Could you please also add a link to the Scanner

Reply:
+1

Reply:
I added a link to vendor and to the documentation.